src/components/App.vue

```diff
@@ -17,11 +17,25 @@
           </div>
 
           <div class="col-12">
-            <input-text v-model="title" title="Issue Title" required autofocus/>
+            <input-text v-model="title" title="Issue Title" @blur="findIssues" required autofocus subtitle="yes">
+              <template slot="subtitle">
+              <div class="similar-issues" v-if="issues.length">
+                Similar issues:
+
+                <ul>
+                  <li v-for="issue in issues.slice(0, 5)" :key="issue.id">
+                    <a class="issue" :href="issue.url" target="_blank" rel="noreferrer" tabindex="-1">
+                      {{ issue.title }}
+                    </a>
+                  </li>
+                </ul>
```

Comment:
And maybe a show more action link here? With `issues.slice(0, (showMoreSimilarIssues && 25) || 5)`